src/main/java/Information.java

```diff
@@ -0,0 +1,4 @@
+public class Information {
+    Task List[] = new Task[100];
+    int listLength = 0;
```

Comment:
Should both of these variables be static as there's only a single instance of them through the whole program?